Makefile

```diff
@@ -1,6 +1,8 @@
 # PREFIX should be only writable by the root to avoid privilege escalation with launchd or sudo
 PREFIX ?= /opt/socket_vmnet
 
+DEBUG ?=
```

Comment:
I would add a warning this this is known to break reproducible builds.

Reply:
done